components/server/src/orgs/organization-service.spec.db.ts

```diff
@@ -55,7 +41,38 @@ describe("OrganizationService", async () => {
         const typeorm = container.get(TypeORM);
         const dbConn = await typeorm.getConnection();
         await dbConn.getRepository(DBTeam).delete({});
-        await (await typeorm.getConnection()).getRepository(DBUser).delete(owner.id);
+        const repo = (await typeorm.getConnection()).getRepository(DBUser);
+        await repo.delete(owner.id);
+        await repo.delete(member.id);
+        await repo.delete(stranger.id);
+    });
+
+    it("should allow only owners to an org", async () => {
```

Comment:
FWIW, this is probably the most significant improvement to the server code-base in a while - the ability to write tests for these services and actually assert on the data level.